### ISSUE TITLE
Create an author ID for Darlene Kwamboka

### ISSUE THREAD

**Issue by COpenda** [onaio/blog#144]
Bio: "Darlene is a data analyst intern at Ona"

**Comment by rowo** [comment]
Is this tied to a specific post?

**Comment by COpenda** [comment]
Yes, she wrote a post about the World Vision expo they went to in Naivasha

**Comment by rowo** [comment]
Should be included in https://github.com/onaio/blog/pull/146

**Comment by rowo** [comment]
@COpenda you shouldn't close this until it's merged
